opentelemetry-sdk/tests/trace/test_trace.py

```diff
@@ -251,6 +252,7 @@ def test_start_span_explicit(self):
         other_parent = trace_api.SpanContext(
             trace_id=0x000000000000000000000000DEADBEEF,
             span_id=0x00000000DEADBEF0,
+            trace_options=trace_api.TraceOptions.get_sampled(),
```

Comment:
`get_sampled` is being called without arguments only once, here. Instead of having that class function, why not return the casted value of `TraceOptions.SAMPLED` here?